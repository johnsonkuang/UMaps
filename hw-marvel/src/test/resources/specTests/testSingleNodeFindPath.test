# Simple test for finding path to itself
LoadGraph graph singleNode.tsv

ListChildren graph 1

FindPath graph 1 1