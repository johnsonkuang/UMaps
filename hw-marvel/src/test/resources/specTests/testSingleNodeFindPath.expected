# Simple test for finding path to itself
loaded graph graph

the children of 1 in graph are:

path from 1 to 1: